File: Programming EF Code First/chapter3/Chapter3/BAGA_DataAnnotations/Model/Lodging.cs
using System.ComponentModel.DataAnnotations;

namespace Model
{
  public class Lodging
  {
    public int LodgingId { get; set; }
    [Required]
    [MaxLength(200)]
    [MinLength(10)]
    public string Name { get; set; }
    public string Owner { get; set; }
    public bool IsResort { get; set; }
    public decimal MilesFromNearestAirport { get; set; }

    public Destination Destination { get; set; }
  }
}

File: Programming EF Code First/chapter3/Chapter 3 VB/Data Annotations/Model/Lodging.vb
Imports System.ComponentModel.DataAnnotations

Namespace Model
  Public Class Lodging
	Public Property LodgingId() As Integer
	<Required, MaxLength(200), MinLength(10)>
	Public Property Name() As String
	Public Property Owner() As String
	Public Property IsResort() As Boolean
	Public Property MilesFromNearestAirport() As Decimal

	Public Property Destination() As Destination
  End Class
End Namespace